### ISSUE TITLE
[BUG] Crafting Table Issue

### ISSUE THREAD

**Issue by CyrusMancini** [ParadoxGamingRP/bugs-and-suggestions#973]
### Your Discord Name

JanethantheNerd#3159

### A clear and concise description of what the bug is

I am unable to use the crafting tables. We were given the blueprints for the Heavy Armor and since then I haven't been able to use the crafting bench. Other people seem to be able to use the table, but myself and the others who received that blueprint haven't been able to use it as far as I know.

### Steps to Reproduce the Bug

Try to use the crafting table after learning the Heavy Armor blueprint

### Expected Behavior

To use the crafting table to craft all the blue prints that I have learned, including the Heavy Armor blueprint. 

### Screenshots/Videos

[https://cdn.discordapp.com/attachments/856710271933218817/1052321991782309888/Crafting_Bench.mp4](url)

### Log File

[CitizenFX_log_2022-12-13T203336.log](https://github.com/ParadoxGamingRP/bugs-and-suggestions/files/10221906/CitizenFX_log_2022-12-13T203336.log)

**Comment by JnKTechstuff** [comment]
You gave me a log where you didn't even connect

**Comment by CyrusMancini** [comment]
[CitizenFX_log_2022-12-14T234817.log](https://github.com/ParadoxGamingRP/bugs-and-suggestions/files/10232576/CitizenFX_log_2022-12-14T234817.log)

Sorry about that. Here is the log I just made. I tested the crafting bench again at the very end. I also took a screenshot of the script right after I used the bench before I logged out if this helps. Thanks for taking a look for us!

![image](https://user-images.githubusercontent.com/120516670/207742328-f20fda6c-24c0-4d68-ad1f-421f3e8defc7.png)
